perf/benchmark/flame/palette.map

```diff
@@ -1,156 +1,148 @@
-std::vector<std::shared_ptr<Envoy::Stats::Counter>, std::allocator<std::shared_ptr<Envoy::Stats::Counter> > >::~vector()->rgb(206,174,54)
-std::vector<std::shared_ptr<Envoy::Stats::Gauge>, std::allocator<std::shared_ptr<Envoy::Stats::Gauge> > >::~vector()->rgb(243,221,27)
-sys_epoll_wait->rgb(245,147,43)
-syscall->rgb(216,4,18)
-tc_deletearray_nothrow->rgb(223,47,16)
-tc_newarray->rgb(251,160,45)
-tcmalloc::CentralFreeList::FetchFromOneSpans(int, void**, void**)->rgb(227,145,37)
-tcmalloc::CentralFreeList::FetchFromOneSpansSafe(int, void**, void**)->rgb(216,23,42)
-tcmalloc::CentralFreeList::InsertRange(void*, void*, int)->rgb(218,6,26)
-tcmalloc::CentralFreeList::ReleaseToSpans(void*)->rgb(238,193,23)
-tcmalloc::CentralFreeList::RemoveRange(void**, void**, int)->rgb(239,159,53)
-tcmalloc::ThreadCache::FetchFromCentralCache(unsigned int, int, void* (*)(unsigned long))->rgb(218,168,22)
-tcmalloc::ThreadCache::ListTooLong(tcmalloc::ThreadCache::FreeList*, unsigned int)->rgb(234,88,25)
-tcmalloc::ThreadCache::ReleaseToCentralCache(tcmalloc::ThreadCache::FreeList*, unsigned int, int)->rgb(219,13,38)
-udp_manip_pkt->rgb(234,134,3)
-udp_send_skb->rgb(245,210,13)
-udp_sendmsg->rgb(236,197,29)
-validate_xmit_skb->rgb(217,154,36)
-validate_xmit_skb_list->rgb(221,47,49)
-veth_xmit->rgb(251,117,31)
-virtqueue_add_outbuf->rgb(226,204,47)
-virtqueue_get_buf_ctx->rgb(209,113,45)
-void fmt::BasicWriter<char>::write_int<unsigned long long, fmt::FormatSpec>(unsigned long long, fmt::FormatSpec)->rgb(251,223,41)
-void fmt::internal::format_decimal<unsigned long, char, fmt::internal::NoThousandsSep>(char*, unsigned long, unsigned int, fmt::internal::NoThousandsSep) (.isra.114)->rgb(208,143,32)
-void std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_construct<char const*>(char const*, char const*, std::forward_iterator_tag) (.isra.148)->rgb(223,46,49)
+->rgb(231,35,9)
```

Comment:
I think you do not need to upload palette.map file. Each run should generate a difference palette.map file. This is not necessary to include in your commit.